### ISSUE TITLE
tests qol: add some warnings filters

### ISSUE THREAD

**Issue by richardjgowers** [OpenFreeEnergy/openfe#415]
we currently get ~95 warnings from the `openfe test` output.  Many of these are about we're renaming molecules, which isn't worth putting on screen, and might also be hiding warnings we should care about...

**Comment by dwhswenson** [comment]
Yeah, I was ranting yesterday about how we also have a bunch that are generated in fixtures (because we're using molecules without hydrogens to make some tests more readable). I've tried to silence warnings in fixtures -- that apparently multiplies them. Looks like we need to use `pytest.warningsfilter` wherever we use fixtures that don't have hydrogens.
